addons/point_of_sale/static/src/app/components/orderline/orderline.xml

```diff
@@ -9,7 +9,7 @@
             </div>
             <div class="d-flex flex-column w-100 gap-1">
                 <div class="d-flex justify-content-between">
-                    <div class="product-name d-inline-block flex-grow-1 fw-bolder pe-1 text-truncate">
+                    <div class="product-name d-inline-block flex-grow-1 fw-bolder pe-1 text-truncate" t-attf-class="{{ props.highlightRefund &amp;&amp; line.isRefund ? 'text-info' : '' }}">
```

Comment:
```suggestion
                    <div class="product-name d-inline-block flex-grow-1 fw-bolder pe-1 text-truncate" t-att-class="{ 'text-info': props.highlightRefund and line.isRefund }">
```

I'd prefer t-att-class which accepts an object of `{ [key as string]: boolean }`, such that if the `key` points to `true`, then it's added in the class list.

Reply:
You're right, it's better.